doc/todo/BACKEND

```diff
@@ -0,0 +1,23 @@
+- documentation of various new functions and the new struct _BackendData
```

Comment:
Can you improve this a bit so that it gives a better overview about:

- what do you plan to do?
- what is already assigned to someone else?
- where do we already have issues?

Reply:
Actually I didn't know this file existed. AFAICT it's still from #2969. I'll just remove the file. The big things are in #4407 and there are lots of `TODO`s throughout the repo (which I'll change to `TODO [new_backend]` for easy grepping).